nototools/android_patches.py

```diff
@@ -0,0 +1,431 @@
+#!/usr/bin/python
+# -*- coding: UTF-8 -*-
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Patches for Android versions of Noto fonts."""
+
+import glob
+from os import path
+
+from nototools import subset
+from nototools import coverage
+from nototools import fix_khmer_and_lao_coverage as merger
```

Comment:
Probably a good idea to move merge_chars_from_bank from fix_khmer... to some other module that both scripts can share.